lms/envs/common.py

```diff
@@ -1852,6 +1852,11 @@
 # parallel, and what the SES rate is.
 BULK_EMAIL_RETRY_DELAY_BETWEEN_SENDS = 0.02
 
+############################# Persistent Grades ####################################
+
+# Queue to use for updating persistent grades
+RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
By defining this value to be `LOW_PRIORITY_QUEUE` here, we leave open the option of creating a specialized grading queue in the future.